src/ext/mpack/mpack.h

```diff
@@ -0,0 +1,7151 @@
+/**
```

Comment:
Would it make sense to also bundle other filfes from the amalgamation package?
```
AUTHORS.md
CHANGELOG.md
LICENSE
README.md
``` <- Those file would be a good reference to have even if the author doesn't require them to be included

Reply:
Great point! I'll pull these in! :)